test/unit/iterator/src/ConstantIterator.cpp

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2021 Anton Reinhard
+ *
+ * This file is part of vikunja.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <vikunja/mem/iterator/ConstantIterator.hpp>
+
+#include <alpaka/alpaka.hpp>
+
+#include <catch2/catch.hpp>
+
+using namespace vikunja::mem::iterator;
```

Comment:
`using namespace` is no good practices, except for literals in local scopes (and maybe other really rarely use case). 
This video explains why: https://youtu.be/MZqjl9HEPZ8

You should use `using vikunja::mem::iterator::ConstantIterator`.